core/sis-feature/src/test/java/org/apache/sis/feature/builder/FeatureTypeBuilderTest.java

```diff
@@ -344,4 +347,95 @@ private static void assertPropertyEquals(final String name, final Class<?> value
         }
         assertEquals("valueClass", valueClass, ((DefaultAttributeType<?>) property).getValueClass());
     }
+
+    @Ignore("Obviously a bug.")
+    @Test
+    public void testToStringInternalIsNotNull() {
+        FeatureTypeBuilder featureTypeBuilder = new FeatureTypeBuilder(null);
+        DefaultFeatureType[] defaultFeatureTypeArray = new DefaultFeatureType[6];
+        featureTypeBuilder.setSuperTypes(defaultFeatureTypeArray);
+        StringBuilder stringBuilder = new StringBuilder("");
+        featureTypeBuilder.toStringInternal(stringBuilder);
+
+        assertNotNull(stringBuilder.toString());
+    }
+
+    @Test
+    public void testBuildThrowsRuntimeException() {
+        FeatureTypeBuilder featureTypeBuilder = new FeatureTypeBuilder();
+        FeatureTypeBuilder featureTypeBuilderTwo = featureTypeBuilder.setIdentifierDelimiters("[RML1*!JGC`fF+|u", "[RML1*!JGC`fF+|u", "[RML1*!JGC`fF+|u");
```

Comment:
It is better to use realistic test case. The `"[RML1*!JGCfF+|u"` separator has no meaning...